include/abb_librws/rws_resource.h

```diff
@@ -26,6 +34,22 @@ namespace abb :: rws
      * \brief IO signal name.
      */
     std::string name;
+
+    bool equals(const SubscribableResource& rhs) const override
+    {
+      if (const IOSignalResource* d = dynamic_cast<const IOSignalResource*>(&rhs); d != nullptr)
```

Comment:
Do you know that you can write simply
```suggestion
      if (const IOSignalResource* d = dynamic_cast<const IOSignalResource*>(&rhs))
```